### DISCUSSION TITLE
How to limit length for input editor

### DISCUSSION THREAD

**Discussion by hauphvn** [megahertz/react-simple-wysiwyg#29] [Q&A]
Hi all,

How do I check number of letter typying into  the editor?

Thanks,

**Comment by megahertz** [comment]
You can calculate the length of the text in the `onChange` handler

**Reply by hauphvn** [reply]
It is not  my expected.
Example: 
<b>test</b>
Actual: length: 11
Expected: length: 4

**Comment by hauphvn** [comment]
Thank you, I have solved that issue by using regex html code and replace on onChange.

Thank you very much. @megahertz
